### PR TITLE
Update/pepe hmstr oraichain

### PR DESCRIPTION
**Pull request by haunv3**

### CODE REVIEW

File: oraichain/assetlist.json [hunk 1]
```diff
@@ -720,8 +718,100 @@
...
+          "counterparty": {
+            "chain_name": "ton",
+            "base_denom": "nanoHmstr"
+          },
+          "provider": "Oraichain Labs Ton Bridge"
+        }
+      ],
+      "base": "factory/orai1wuvhex9xqs3r539mvc6mtm7n20fcj3qr2m0y9khx6n5vtlngfzes3k0rq9/HMSTR",
+      "name": "HMSTR",
+      "display": "hmstr",
```
Comment (JeremyParish69) [inline oraichain/assetlist.json:793]: this denom (case sensitive) isn't defined in denom_units. Either update this to "HMSTR" to make the defined unit, or change the exponent:9 denom unit to "hmstr" so it matches this.

File: oraichain/assetlist.json [hunk 2]
```diff
@@ -720,8 +718,100 @@
...
+      "symbol": "HMSTR",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/hmstr.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/hmstr.svg"
+      },
+      "images": [
+        {
+          "image_sync": {
+            "chain_name": "ton",
+            "base_denom": "nanoHmstr"
```
Comment (JeremyParish69) [inline oraichain/assetlist.json:803]: This asset isn't defined, so there's nowhere to pull the image from. Can we register this asset as well under TON's assetlist?

Comment (haunv3) [reply oraichain/assetlist.json:803]: This is an asset on the TON network. You can register it. 👍

File: oraichain/assetlist.json [hunk 3]
```diff
@@ -720,8 +718,100 @@
...
+          },
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/hmstr.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/oraichain/images/hmstr.svg",
```
Comment (JeremyParish69) [inline oraichain/assetlist.json:806]: This SVG file size is way too large. Please only include files <250kB.

**Review by JeremyParish69 — CHANGES_REQUESTED** [review]
Display denom is not correct. Image file size is too large. Referenced asset is not registered.

**Comment by JeremyParish69** [comment]
Appears abandoned

**Pull request closed without merging**